public/app/features/explore/utils/links.ts

```diff
@@ -146,19 +132,34 @@ export const getFieldLinksForExplore = (options: {
           });
         }
 
-        return mapInternalLinkToExplore({
-          link,
-          internalLink: link.internal,
-          scopedVars: { ...scopedVars, ...internalLinkSpecificVars },
-          range,
-          field,
-          onClickFn: splitOpenFn,
-          replaceVariables: getTemplateSrv().replace.bind(getTemplateSrv()),
-        });
+        const allVars = { ...scopedVars, ...internalLinkSpecificVars };
+        const variableData = getVariableUsageInfo(link, allVars);
+        let variables: VariableInterpolation[] = [];
+        if (Array.from(variableData.variables.keys()).length === 0) {
```

Comment:
Do we need `Array.from` here?

```suggestion
        if (variableData.variables.length === 0) {
```

Reply:
@ifrost nope, and having this meant the key wasnt showing in static links. I fixed it and added a comment on what that code is for.

Reply:
@ifrost nope, and having this meant the key wasnt showing in static links. I fixed it and added a comment on what that code is for.  Will add a test next.